bdd/step-definitions/e2e/cli.ts

```diff
@@ -7,10 +7,16 @@ import * as fs from "fs";
 import { getStreamsFromSpawn } from "../../lib/utils";
 import { expectedResponses } from "./expectedResponses";
 
-const si = process.env.SCRAMJET_SPAWN_TS
-    ? ["npx", "ts-node", "../packages/cli/src/bin/index.ts"]
-    : ["node", "../dist/cli/bin"]
+// const si = process.env.SCRAMJET_SPAWN_TS
+//     ? ["npx", "ts-node", "../packages/cli/src/bin/index.ts"]
+//     : ["node", "../dist/cli/bin"]
+// ;
+
+const si = process.env.SCRAMJET_SPAWN_JS
```

Comment:
I'd add it to our ENV Bible: https://github.com/scramjetorg/transform-hub/blob/release/0.13/ENV_VARS.md :)
so the way I understand it, our tests which are not run with `SCRAMJET_SPAWN_JS` will be testing the CLI installed from npm (the one installed locally)?
Hmm maybe we could mention it somewhere in the tests. I just feel I will find myself debugging the new fixes in CLI and not understanding why they still don't work :smile:

Reply:
as an alternative perhaps allowing one of the three: ts/js/npm?

Reply:
new ENV added to our Bible 😉 I also added short description which says that when `SCRAMJET_SPAWN_JS` is set to `false` cli commands are executed from npm package with `si` command.

Reply:
excellent, thanks :)